packages/aws-cdk/README.md

```diff
@@ -375,15 +375,20 @@ $ cdk deploy --hotswap [StackNames]
 ```
 
 This will attempt to perform a faster, short-circuit deployment if possible
-(for example, if you only changed the code of a Lambda function in your CDK app,
-but nothing else in your CDK code),
+(for example, if you changed the code of a Lambda function in your CDK app),
 skipping CloudFormation, and updating the affected resources directly;
 this includes changes to resources in nested stacks.
 If the tool detects that the change does not support hotswapping,
-it will fall back and perform a full CloudFormation deployment,
-exactly like `cdk deploy` does without the `--hotswap` flag.
+it will ignore it and display that ignored change.
+To have hotswap fall back and perform a full CloudFormation deployment,
+exactly like `cdk deploy` does without the `--hotswap` flag,
+specify `--hotswap-fallback`, like so:
```

Comment:
Maybe a name along this line is better?
```suggestion
specify `--hotswap-all`, like so:
```

Reply:
it's a bit misleading, because it implies that all the resources will be hotswapped, when they'll be deployed via CFN. We could be really verbose and do `--hotswap-fallback-to-cfn`

Reply:
Maybe `--hotswap-only` could mean only update the hot-swappable things. Adding a new flag like that instead of changing the behavior of the existing flag could prevent breaking all the scripts that are already using that flag. It also allows the [documentation](https://docs.aws.amazon.com/cdk/v2/guide/cli.html) to remain correct.

And `--with-hotswap` could be added as an alias for `--hotswap` if the original flag name is not clear enough.